docs/LV_client/examples/mono3_replay_example.md

```diff
@@ -0,0 +1,52 @@
+# mono3_replay_example.vi
+
+<p class="img_container">
+<img class="lg_img" src="../mono3_replay_example.png"/>
+</p>
+
+### Description
+
+This example shows how to run the monoDrive client to configure the monoDrive simulator on Replay.
+
+For technical support contact us at <b>support@monodrive.io</b> 
+
+### Inputs
+
+- **Trajectory Configuration:**  Selected a valid scenario file in JSON. Use the browse
```

Comment:
Continue to use the active voice on all input keys: "Select" instead of "Selected"